Assets/StreamingAssets/Localization/en_US.lang

```diff
@@ -82,14 +80,16 @@ tile_Floor=Floor
 tile_Floor_desc=Meets the highest non-skid standards.
 tile_Ladder=Ladder
 tile_Ladder_desc=High-tech space movement system. Effectiveness reduced when held by weak noodle arms.
+roombehavior_airlock=Airlock
```

Comment:
I think room behavior is part of your other PR, but considering this one is big, it will likely be in master before this gets merged.